File: web_shop/src/main/webapp/js/service/orderService.js
//服务层
app.service('orderService',function($http){

    //搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../order/search.do?page='+page+"&rows="+rows,searchEntity);
    }
    //审核状态
    this.updateStatus = function(ids,status){
        return $http.get('../order/updateStatus.do?ids='+ids+"&status="+status);
    }
});

File: interface/src/main/java/cn/itcast/core/service/OrderService.java
package cn.itcast.core.service;


import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.order.Order;
import cn.itcast.core.pojo.order.OrderItem;
import org.springframework.web.bind.annotation.RequestBody;

import java.util.List;

import java.util.Map;

public interface OrderService {
    void add(Order order, String userName);

    Map<String,Double> findSaleData(String start, String end, String sellerId) ;
    public PageResult search(Integer page, Integer rows, Order order );
    public void updateStatus(Long [] ids,String status);
}
